sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/SQLViewSuite.scala

```diff
@@ -543,4 +545,34 @@ class SQLViewSuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
       }
     }
   }
+
+  test("create a nested view") {
```

Comment:
It's not easy to test the `resolveRelations` rule directly in `AnalysisSuite`, so we test resolve a nested view here instead.